### ISSUE TITLE
Team update - Aug 24, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#155]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

* [Rendered notebook user testing - Round 1](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/146)

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf for letting me work with her on an interesting blog post
- @steff456 for agreeing to help @kulsoomzahra navigate the zoom-related issues in JupyterLab
- @trallard for keeping eyes and ears open for new opportunities to bring more support to this team's work (Berkeley)

**Updates from last week :heavy_check_mark:**
- https://github.com/jupyterlab/lumino/pull/373. There are still [other tab traps that need fixing](https://github.com/jupyterlab/lumino/pull/373#issuecomment-1224098336) in JupyterLab, but the one in the Lumino menu bar was probably the worst offender. I'm not sure when this will get rolled out to end users, though — need to check with Darian or Frederic.
- https://github.com/Quansight-Labs/jupyter-a11y-testing/issues/13. I've branched locally and started writing (refactoring) code, but I won't be able to finish this until next week. I will get in touch with @tonyfast next week if I have any questions or hit any walls.
- Had Q&A with Tania, Isabela, and Berkeley's dev team about the state of accessibility in JupyterLab
- Had chat with Kulsoom about which accessibility issues she wants to work on. She will let you know in her update.

**My availability for next week**
- I'm off Thursday-Monday, August 25-29. 
- I will be working from plane (flying to the USA) on Wednesday, August 31.

**Comment by steff456** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf for always being super helpful

**Updates from last week :heavy_check_mark:**
- I worked in the blurry python 3 icon present in the Launcher

**Challenges I faced and things I'd like assistance with 🙏**
- I'm having a hard time finding the actual asset for the icon, I decided to create an issue to seek for some guidance: https://github.com/jupyterlab/jupyterlab/issues/12997.
- Start organizing future tasks for fixing Zoom issues

**My availability for next week**
- I'll be here!

**Important items for discussion 💬**
- Future zoom related tasks

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**

- #146. User tests are happening! I don't have any documented updates while we're in the middle of it, but I am excited to report back when we get there. 
- #132. Made some changes to the [draft event content](https://github.com/isabela-pf/a11y-events/pull/8) based on feedback, but it's still in draft. I still need to make some slides before we can really test it, but I'd like to schedule something so I have a deadline.
- #133. Has been on the back burner between testing time needed and some napari deadlines. This is going to be my main focus directly after this meeting. I did throw out one scoped solution for tabs on Slack that I'm willing to discuss in more detail if it's timely for anyone.

**Challenges I faced and things I'd like assistance with 🙏**
- I have nothing to talk about.

**My availability for next week**
- I'll be out of office September 1–September 5.

**Important items for discussion 💬**
- Schedule event test run so that I have a deadline? Because I operate on deadlines? Maybe? 😅

**Comment by kulsoomzahra** [comment]
**Thanks I'd like to give 🙌**
- @tonyfast for demonstrating an overview of the many _kind_ of issues on the project board
- @gabalafou for helping me explore the areas where I could potentially work
- @trallard @steff456 @isabela-pf for sharing resources and their thoughts

**Updates from last week :heavy_check_mark:**
- Spent some time figuring out what I could work on and ended up with the Zoom issue
- Tried understanding the [Reflow criterion (1.4.10) of WCAG 2.1](https://drive.google.com/file/d/1VYgRrtCckUMAj_3A21FY3DPei0YYtxd5/view?usp=sharing_eil&ts=5e177beb) and also [Understanding Success Criterion](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

**Challenges I faced and things I'd like assistance with 🙏**
- Nothing at the moment, thanks 

**My availability for next week**
- I'll be here

**Items for discussion 💬**
- Something that quickly came up to my mind when we were thinking of solutions to items of the menu bar was:
1. Add a button that says **Menu**,  expanding on which gives all the menu items
2. Second Menu bar like two rows so that all items of the menu bar are visible (I'm not sure if this aligns with the guidelines, so I'll need to read about it)
-

**Comment by trallard** [comment]
@kulsoomzahra 
- Feels like colour contrast and zoom issues are something she could tackle -> sync with @steff456 

@isabela-pf 
- Has been working on user tests (STScI): two left (this and the upcoming week)
- Will try and make some progress on the reflow mocks: will work on her own sketches
  - there are still plenty of unknowns 
  - @trallard  was reading this https://stephaniewalter.design/blog/a-designers-guide-to-documenting-accessibility-user-interactions/
  - 12th September - team's rundown of the event

@steff456 
1. Make a PR to ipykernel
2. Right now, and apparently only supports png (i.e. 64x64)
3. Will sync with Matthias on how to submit this change
4. Started organizing future tasks for fixing Zoom issues
  - @isabela-pf  mentioned TABS-related fixes could be well scoped for Kulsoom
  - skiplink stuff: there are notes and should also be well scoped


📓 Misc notes:
- Related to zoom: how is the experience related to code wrapping in 400% zoom
